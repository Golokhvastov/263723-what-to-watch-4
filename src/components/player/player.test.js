import React from "react";
import renderer from "react-test-renderer";
import Player from "./player.jsx";

it(`Player render correctly`, () => {
  const tree = renderer
    .create(
        <Player>
          <div></div>
        </Player>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
