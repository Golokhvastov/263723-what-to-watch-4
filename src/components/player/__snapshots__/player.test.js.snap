// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player render correctly 1`] = `<div />`;
